app/views/completed_transaction/_assisted_digital_satisfaction_survey.html.erb

```diff
@@ -0,0 +1,163 @@
+    <br />
+    <label class="block-label">
+      <input type="radio" required name="service_feedback[assistance_received]" id="no-help" value="no">
+      No
+    </label>
+  </fieldset>
+
+  <br/>
+
+  <div id="assistance-yes-section" class="js-hidden">
+    <fieldset>
+      <legend><h2>What assistance did you receive?</h2></legend>
+      <br />
+      <label for="assistance-received-comments" class="visuallyhidden">Your comments</label>
+      <textarea name="service_feedback[assistance_received_comments]" class="full-size counted" id="assistance-received-comments"></textarea>
+      <p id="assistance-received-commentscounter" class="hint" aria-live="polite" aria-atomic="false">(Limit is 1200 characters)</p>
+    </fieldset>
+
+    <br/>
+
+    <fieldset>
+      <legend><h2>Who provided the assistance?</h2></legend>
+      <br />
+      <label class="block-label">
+        <input type="radio" required name="service_feedback[assistance_provided_by]" id="friend-relative" value="friend-relative">
+        A friend or relative
+      </label>
+      <br />
+      <label class="block-label">
+        <input type="radio" required name="service_feedback[assistance_provided_by]" id="work-colleague" value="work-colleague">
+        A work colleague
+      </label>
+      <br />
+      <label class="block-label" data-target="assistance-staff-or-other-section">
+        <input type="radio" required name="service_feedback[assistance_provided_by]" id="government-staff" value="government-staff">
+        A staff member of the responsible government department
+      </label>
+      <br />
+      <label class="block-label" data-target="assistance-staff-or-other-section">
+        <input type="radio" required name="service_feedback[assistance_provided_by]" id="other" value="other">
+        Other (please specify)
+      </label>
+      <br/>
+
+      <label for="other-specified" class="visually-hidden">Tell us who the other person was</label>
+      <input type="text" name="service_feedback[assistance_provided_by_other]" id="other-specified" value="">
+    </fieldset>
+
+    <br />
+
+    <div id="assistance-staff-or-other-section" class="js-hidden">
+      <fieldset>
+        <legend><h2>How satisfied are you with the assistance received?</h2></legend>
+        <br />
+        <label class="block-label">
+          <input type="radio" required name="service_feedback[assistance_satisfaction_rating]" id="assistance-very-satisfied" value="5">
+          Very satisfied
+        </label>
+        <br />
+        <label class="block-label">
+          <input type="radio" required name="service_feedback[assistance_satisfaction_rating]" id="assistance-satisfied" value="4">
+          Satisfied
+        </label>
+        <br />
+        <label class="block-label">
+          <input type="radio" required name="service_feedback[assistance_satisfaction_rating]" id="assistance-neither-satisfied-or-dissatisfied" value="3">
+          Neither satisfied or dissatisfied
```

Comment:
I'll get a steer from a content person as this is the wording that's been here since time immemorial.